processes.go

```diff
@@ -129,7 +159,7 @@ func (cm CommandMap) Value() (driver.Value, error) {
 	for k, v := range cm {
 		m[string(k)] = sql.NullString{
 			Valid:  true,
-			String: string(v),
+			String: v.String(),
```

Comment:
I think right now, it would only be a problem during a rollback, since that's the only time the slug would actually be loaded from the database.